src/query/interpreter.cpp

```diff
@@ -1004,15 +1004,12 @@ Callback HandleReplicationQuery(ReplicationQuery *repl_query, const Parameters &
   evaluation_context.timestamp = QueryTimestamp();
   evaluation_context.parameters = parameters;
   auto evaluator = PrimitiveLiteralExpressionEvaluator{evaluation_context};
-
+  auto const is_coordinator_managed = flags::CoordinationSetupInstance().IsCoordinatorManaged();
```

Comment:
Can we rename IsCoordinatorManaged to IsManagedByCoordinator? 
I have a feeling that I am reading if coordinator is being managed instead of something else